docs/source/conf.py

```diff
@@ -96,7 +96,7 @@
     "learn/install": "../install.html",
     "learn/user_guides/dataframes": "intro-dataframes.html",
     "learn/user_guides/types_and_ops": "intro-dataframes.html",
-    "learn/user_guides/remote_cluster_execution": "distributed-computing.html",
+    "learn/user_guides/remote_cluster_execution": "user_guide/distributed-computing",
```

Comment:
We should also add a redirect from the `poweruser/distributed-computing` page to `distributed-computing` I believe